app/assets/stylesheets/components/buttons.scss

```diff
@@ -427,7 +427,6 @@ label[class*='crayons-btn']:focus-within // label for input[type="file"] made to
   font: inherit;
   text-align: center;
   cursor: pointer;
-  transition: all var(--transition-props);
```

Comment:
Decided to remove transitions as they don't add too much to the entire experience (its like 100ms) and they make things look actually a bit weird and choppy while navigating with keyboard.

I did the same for CTAs and links.